### PR TITLE
8wA933sS: Add Acceptance tests

### PR DESCRIPTION
**Pull request by kerrr**
* Replaced Home view and Controller for a stable entry point for testing.
* Added Rake task for acceptance tests.
* Added basic acceptance test that visits the Home page.
* Added acceptance helper to configure the acceptance test environment.
* Filtered out acceptance tests so they do not run under when not required.
* Replaced system test for home page.

Co-authored-by: Rosa Fox <rosa.fox@digital.cabinet-office.gov.uk>
Co-authored-by: Olakunle Jegede <olakunle.jegede@digital.cabinet-office.gov.uk>

This is a new branch with all the work from the old one on top of master after my git-fu spectacularly failed me.

### CODE REVIEW

File: app/helpers/home_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (Rosa-Fox) [inline app/helpers/home_helper.rb]: This is unused so we could get rid of it. Guessing Rails generated it.

File: spec/acceptance/visit_home_controller_acceptance_spec.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'acceptance_helper'
+
+RSpec.describe 'Homepage', type: :feature, acceptance: true do
+
+  it 'shows greeting with JS', js: true do
+    visit "https://#{ENV["TEST_DOMAIN"]}/home/index"
+    #visit "https://verify-self-service-dev.cloudapps.digital/home/index"
```
Comment (Rosa-Fox) [inline spec/acceptance/visit_home_controller_acceptance_spec.rb]: Comment can be removed.

File: spec/acceptance_helper.rb [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+end
+
+RSpec.configure do |config|
+  # rspec-expectations config goes here. You can use an alternate
+  # assertion/expectation library such as wrong or the stdlib/minitest
+  # assertions if you prefer.
+  config.filter_run_excluding :acceptance => true
+
+  if config.filter_manager.inclusions[:acceptance]
```
Comment (Rosa-Fox) [inline spec/acceptance_helper.rb]: We can remove this `if` statement.

File: app/helpers/home_helper.rb (hunk 1 above)
Comment (kerrr) [reply app/helpers/home_helper.rb]: Yeah, not sure what those do. There are three of them, all doing nothing.  I'd prefer to just leave them for now and then look again when we do some tidying up.

**Pull request merged**